solr/solr-ref-guide/src/collection-management.adoc

```diff
@@ -350,31 +506,46 @@ update commands, which are directed now to `collection2`.
 [[delete]]
 == DELETE: Delete a Collection
 
-`/admin/collections?action=DELETE&name=_collection_`
+The DELETE action is used to delete a collection.
+
+[.dynamic-tabs]
+--
+[example.tab-pane#v1deletecollection]
+====
+[.tab-label]*V1 API*
+
+With the v1 API, the `delete` command must be capitalized as `DELETE`:
+
+[source,bash]
+----
+http://localhost:8983/solr/admin/collections?action=DELETE&name=newCollection
+----
+====
+
+[example.tab-pane#v2deletecollection]
+====
+[.tab-label]*V2 API*
+
+
+[source,bash]
+----
+curl -X DELETE http://localhost:8983/api/collections/newCollection?omitHeader=true
+----
+====
+--
 
 === DELETE Parameters
 
 `name`::
 The name of the collection to delete. This parameter is required.
 
 `async`::
-Request ID to track this action which will be <<collections-api.adoc#asynchronous-calls,processed asynchronously>>.
+Request ID to track this action which will be <<collections-api.adoc#asynchronous-calls,processed asynchronously>> and is a V1 API only parameter.
```

Comment:
I wonder if this was done intentionally or is just something that Noble or whoever forgot to put in the apispec file.  It's a real shame to have to document stuff that's almost surely just accidental oversight. 🤷 

But I respect that you don't want to bundle in any actual API changes to this PR.

Reply:
This is a great point, and I hadn't thoguht about it till you mentioned it, but in my quick look, maybe it's just missing.  I'm going to go see if I can fix SOLR-15162 to have the async...

Reply:
Really appreicate your digging through this on a detailed level!

Reply:
Okay, I've shot an hour messing around with variations on `collections.collection.delete.json` etc with no luck...    And I want to get back to this.   @noblepaul or @gerlowskija any chance you could look at the fix in SOLR-15162.   I am going to edit back out the caveat about the V1 api on the idea that it will get fixed ;-)